restcomm/restcomm.testsuite/src/test/resources/restcomm.script-applications

```diff
@@ -0,0 +1,37 @@
+CREATE SCHEMA PUBLIC AUTHORIZATION DBA
```

Comment:
We have been discussing on maintenance cost of hsql scripts needed for tests-suit:
imo we should handle a new test's logic by reusing an existing script and add new rows of our desire.

Reply:
@maria-farooq , i agree in principle. Is this script in place ? There should be one with guarantees that it won't contain any more INSERTS that may affect the results of other tests.

In any case, i consider what you describe a separate issue altogether.